htmresearch/encoders/language_encoder.py

```diff
@@ -183,11 +183,14 @@ def sparseUnion(self, counts):
 
     @param counts     (Counter)   A count of the ON bits for the union bitmap.
 
-    @return           (list)      A sparsified union bitmap.
+    @return           (tuple)      A sparsified union bitmap.
     """
+    positionsSortedByCount = tuple(x[1] for x in counts.getNonZerosSorted())
```

Comment:
Is this a stable sort? In case of ties, does it prefer lower indices?  If so, that could be problematic.